cmd/kube-proxy/app/conntrack.go

```diff
@@ -37,8 +37,14 @@ type Conntracker interface {
 	SetMax(max int) error
 	// SetTCPEstablishedTimeout adjusts nf_conntrack_tcp_timeout_established.
 	SetTCPEstablishedTimeout(seconds int) error
-	// SetTCPCloseWaitTimeout nf_conntrack_tcp_timeout_close_wait.
+	// SetTCPCloseWaitTimeout adjusts nf_conntrack_tcp_timeout_close_wait.
 	SetTCPCloseWaitTimeout(seconds int) error
+	// SetTCPBeLiberal adjusts nf_conntrack_tcp_be_liberal.
+	SetTCPBeLiberal(value int) error
+	// SetUDPTimeout adjusts nf_conntrack_udp_timeout.
+	SetUDPTimeout(seconds int) error
+	// SetUDPStreamTimeout adjusts nf_conntrack_udp_timeout_stream.
+	SetUDPStreamTimeout(seconds int) error
```

Comment:
when is this used?  nf_conntrack_udp_timeout_stream is something I do not have clear when it applies

Reply:
For UDP connections if the data is transferred in both directions, conntrack treats it as udp stream and uses **udp_timeout_stream** instead of **udp_timeout**.

Reply:
I see, I didn't remember that we were setting `nf_conntrack_tcp_timeout_established` too, that is the equivalent of `nf_conntrack_udp_timeout_stream`